cmd/networkcmd/start.go

```diff
@@ -59,7 +59,7 @@ func startNetwork(cmd *cobra.Command, args []string) error {
 
 	ux.Logger.PrintToUser(startMsg)
 
-	outputDirPrefix := path.Join(app.GetRunDir(), "restart")
+	outputDirPrefix := path.Join((*app).GetRunDir(), "restart")
```

Comment:
This does not come from this PR but we should use `filepath.Join`. `path` targets mostly URLs, and can't handle all filesystems correctly.

Reply:
Made a ticket for this: https://ava-labs.atlassian.net/browse/AV-1856